config/owasp/suppressions.xml

```diff
@@ -35,6 +35,7 @@
     <cve>CVE-2022-25647</cve>
     <cve>CVE-2016-1000027</cve>
     <cve>CVE-2022-25647</cve>
+    <cve>CVE-2022-29885</cve>
```

Comment:
Is there any way to update libraries to get rid of this suppression or are we forced to add suppression?

Reply:
@bhssamray i've removed them now